tools/importer/import.js

```diff
@@ -159,6 +159,20 @@ const createMetadata = (main, document, url) => {
     meta.Image = el;
   }
 
+  // Preference for image alt tag is og:image:alt > og:title > title
+  let imagealt = document.querySelector('[property="og:image:alt"]');
+  if (imagealt === null || imagealt === undefined) {
```

Comment:
!imagealt will also work here

Reply:
Right `null` and `undefined` are falsey in javascript so your suggestion would also work.